.github/workflows/linux.yml

```diff
@@ -2,7 +2,7 @@ name: Linux
 on:
   push:
     branches:
-      - "*"
+      - "**"
     tags:
       - "*"
```

Comment:
```yaml
branches:
tags:
```

will work.

Reply:
Oh...I see.

"on.push" event only has branches parameter and tags parameter.
the following definition is the same as "on: push:"

```
on:
  push:
    branches:
    tags:
```

Therefore, I've fixed as below.

```
on:
  push:
```

The following definition is also the same as "pull_request: branches:". 　I also fixed this.